packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -174,6 +189,65 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
 
   // Internal helper methods:
 
+  // Returns an ExpressionPriceFeed.
+  async function _createExpressionPriceFeed(expressionConfig) {
+    // Build list of configs that could be used in the expression including default price feed configs and customFeeds
+    // that the user has provided inside the ExpressionPriceFeed config. Note: default configs are overriden by
+    // customFeeds with the same name. Tranform keys by stripping "/" since that would be interpreted as division.
+    const allConfigs = Object.fromEntries(
+      Object.entries({ ...defaultConfigs, ...expressionConfig.customFeeds }).map(([key, value]) => {
+        return [key.replace("/", ""), value];
+      })
+    );
+
+    // This call chain:
```

Comment:
is there any way to detect if the user has incorrectly placed a `/` within the feed within an expression? I guess if it parses this as deciding two parts of a feed it would error out? ETH/BTC would be ETH divided by BTC and we don't have a feed that is just `ETH` or `BTC` so it would error, right?

Reply:
Exactly. 💯 . That's effectively our only way to verify the correctness -- rely on mathjs to tell us if there are any unresolved symbols.